### DISCUSSION TITLE
open3d.geometry module PointCloud with GPU

### DISCUSSION THREAD

**Discussion by Kedaro** [isl-org/Open3D#4161] [General]
I would like create an instance of PointCloud and compute `remove_radius_outlier` leveraging the GPU.  I have a PointCloud with 150097 points and would like to downsample as little as possible, while still maintaining reasonable latency. Is there a way to use the CUDA with the `open3d.geometry` module, specifically the PointCloud class?

**Comment by ssheorey** [comment]
`open3d.geometry` APIs don't support CUDA, only `open3d.t.` APIs do. This specific function is not implemented with CUDA yet, but you can use this workaround (you will need to use either the PyTorch or TensorFlow version): 
http://www.open3d.org/docs/latest/python_api/open3d.ml.torch.ops.radius_search.html
